inputFiles/solidMechanics/ExtendedDruckerPragerWellbore_benchmark.xml

```diff
@@ -10,32 +10,57 @@
     <InternalWellbore
       name="mesh1"
       elementTypes="{ C3D8 }"
-      radius="{ 0.1, 5.0 }"
-      theta="{ 0, 180 }"
+      radius="{ 0.1, 10.0 }"
+      theta="{ 0, 90 }"
       zCoords="{ -1, 1 }"
       nr="{ 40 }"
-      nt="{ 80 }"
-      nz="{ 2 }"
+      nt="{ 40 }"
+      nz="{ 1 }"
       trajectory="{ { 0.0, 0.0, -1.0 },
                     { 0.0, 0.0, 1.0 } }"
       autoSpaceRadialElems="{ 1 }"
-      useCartesianOuterBoundary="0"
       cellBlockNames="{ cb1 }"/>
   </Mesh>
-
   <!-- SPHINX_DP_WELLBORE_MESH_END -->
   
   <!--Solver application-->
   <Events
-    maxTime="1.0">
+    maxTime="1.05">
     <PeriodicEvent
       name="solverApplications"
+      forceDt="0.5"
```

Comment:
It is better to lower the forceDt here to capture the wellbore shape in the elastic regime.

Reply:
Done. The number of step in the elastic regime was reduced to minimize the size of the hdf5 outputs. Now the hdf5 are removed, the time step in the elastic regime can be reduced.